ReflectionCodingExerciseWeek12/ReflectionCodingExerciseWeek12Test/UnitTest1.cs

```diff
@@ -0,0 +1,127 @@
+﻿using Microsoft.CSharp;
+using System;
+using System.CodeDom;
+using System.Reflection;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using ReflectionCodingExerciseWeek12;
+
+namespace ReflectionCodingExerciseWeek12Test
+{
+    [TestClass]
+    public class UnitTest1
```

Comment:
Per the requirements of the assignment, "The Test project should contain at least one test class for each of the classes in the Class Library". You should have an AirPlaneTests class and a CarTests class, each containing the tests related to those classes. (-0.2)